docs/reference/technology/connectivity/connectivity.md

```diff
@@ -13,66 +13,71 @@ For IP devices:
 - NB-IoT.
 - Bluetooth Low Energy (BLE).
 
-Non-IP devices:
+<!--- Order and headings do not reflect what're in the lists below--->
+Non-IP devices require a gateway:
 
 - LoRaWAN.
 - Cellular.
 
+<!--- What about NFC?--->
+
+<!---What if we move this to the ../technology.md file?--->
```

Comment:
We separated networking architecture docs from non-networking architecture docs. The technology.md file is for both. This content is networking-specific.